### PR TITLE
Hooks

### PR DESCRIPTION
**Pull request by luqmanmusah**
Refacted the Calculator component (components/Calculator.js) from a class-based component to a functional component with hooks.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,9 +6,6 @@ This is the first project of the Math Magicians application. I will set up the e
 
 ## Additions
 
-- Imported the Calculate.js and Operate.js in the Calculator component.
```
Comment (MahmoudBakr23) [inline README.md:9]: - [x] As much as I appreciate you included a picture of your app in the README file, I also highly recommend that you follow this great [template](https://github.com/microverseinc/readme-template) to build up a more consistent and more related to the project README file 😄 🙏

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hello @luqmanmusah 😃, 

Great work and effort you put into this project so far.

## Highlights 🚀 

There are some excellent points that you have made perfectly. Keep up! 💪
- Linter is correct
- Descriptive PR headline, summary, and commit messages

However, there is always room for improvement.

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄

 ## Required Changes ♻️

- [x] Well done refactoring your Calculator component using react hooks, but it also should use arrow function instead of the `function` keyword. It's so important for you to practice the ES6 features in React to get your eyes familiar with it 😉 👍 
- [x] Please double-check on the functionality of the `=` button because it keeps rendering `0` every time I try to get the result of an operation! I recommend that you keep track of the state and how it's changing using `console.log` 
_Please note that the file isn't loaded for inline comments._ 🙏 

_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [x] You followed the design 👏, but isn't it just a little bit bigger! 😅 I highly suggest to make the design more consistent for the looks of your app 😄 🙏 

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by luqmanmusah** [comment]
Fixed all comments mentioned above.
Thanks

**Review by AshakaE — APPROVED** [review]
### Hi @luqmanmusah  🙋‍♀️, 

# STATUS APPROVED ✔
Thanks for making the required changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**